server_admin/topics/authentication/flows.adoc

```diff
@@ -2,197 +2,228 @@
-  evaluation of the `Condition - User Configured` execution. If it is, the executions for this sub-flow are loaded and the same processing logic occurs
-. The next execution is the `Condition - User Configured`. This checks if the other executions in the flow are configured for the user.
-  Meaning that the `Browser - Conditional OTP` sub-flow will only be executed if the user has an OTP credential configured.
-. The final execution is the `OTP Form`. This is marked as _required_, but because of the setup in the _conditional_ subflow, it will only be run if the user
-  has an OTP credential set up. If he doesn't, the user will not see an OTP form.
+====== Required
 
-==== Creating flows
+All _Required_ elements in the flow must be successfully sequentially executed. The flow terminates if a required element fails.
 
-This section explains in greater depth how flows work, and how to create your own flows. Note that there are important functionality and security
-considerations when designing your own flow. A badly created flow could either let no one log in, let users in with less verification than you would
-like, or simply result in an error.
+====== Alternative
 
-To create a flow, you can either:
+Only a single element must successfully execute for the flow to evaluate as successful. Because the _Required_ flow elements are sufficient to mark a flow as successful, any _Alternative_ flow element within a flow containing _Required_ flow elements will not execute.
 
-. Copy and then modify an existing flow. To do this select an existing flow (for example the `Browser` flow), and press the `Copy` button.
-  This will then ask you to set a name for the new flow, before creating it.
-. Create a new flow from scratch. To do this press the `New` button. Since this is the more general case, we will use this for our example.
+====== Disabled
 
-When creating a new flow, you will have to create a top level flow
+The element does not count to mark a flow as successful.
 
-.Create a top level flow
-image:{project_images}/Create-top-level-flow.png[]
+====== Conditional
+
+This requirement type is only set on sub-flows. 
+
+* A _Conditional_ sub-flow contains executions. These executions must evaluate to logical statements. 
+* If all executions evaluate as _true_, the _Conditional_ sub-flow acts as _Required_. 
+* If all executions evaluate as _false_, the _Conditional_ sub-flow acts as _Disabled_.
+* If you do not set an execution, the _Conditional_ sub-flow acts as _Disabled_. 
+* If a flow contains executions and the flow is not set to _Conditional_, {project_name} does not evaluate the executions, and the executions are considered functionally _Disabled_.
+
+==== Creating flows
 
-With the following options:
+There are important functionality and security considerations when designing a flow.
```

Comment:
Important functionality and security considerations apply when you design a flow.